src/Bot.ts

```diff
@@ -1,44 +1,67 @@
 "use strict";
 
+import { calculateBotPick } from "mtgdraftbots";
+
 import { Card } from "./Cards";
 
+const BASICS = [
+	"56719f6a-1a6c-4c0a-8d21-18f7d7350b68",
+	"b2c6aa39-2d2a-459c-a555-fb48ba993373",
+	"bc71ebf6-2056-41f7-be35-b2e5c34afa99",
+	"b34bb2dc-c1af-4d77-b0b3-a0fb342a5fc6",
+	"a3fb7228-e76b-4e96-a40e-20b5fed75685",
+];
+
 export default class Bot {
 	name: string;
 	id: string;
-	cards: Card[] = []; // For debugging mostly.
-	pickedColors: { [color: string]: number } = { W: 0, U: 0, R: 0, B: 0, G: 0 };
+	seen: Card[] = [];
+	cards: Card[] = [];
 
 	constructor(name: string, id: string) {
 		this.name = name;
 		this.id = id; // Used for sorting
 	}
 
-	pick(booster: Card[]) {
-		let maxScore = 0;
-		let bestPick = 0;
-		for (let idx = 0; idx < booster.length; ++idx) {
-			let c = booster[idx];
-			// TODO: Rate cards
-			let score = c.rating;
-			for (let color of c.colors) {
-				score += 0.35 * this.pickedColors[color];
-			}
-			if (score > maxScore) {
-				maxScore = score;
-				bestPick = idx;
-			}
-		}
-		for (let color of booster[bestPick].colors) {
-			this.pickedColors[color] += 1;
-		}
+	getBotResult(booster: Card[], boosterNum: number, numBoosters: number, pickNum: number, numPicks: number) {
+		this.seen.push(...booster);
+		const cardOracleIds = BASICS.concat(
+			booster.map(c => c.oracle_id),
+			this.cards.map(c => c.oracle_id),
+			this.seen.map(c => c.oracle_id)
+		);
+		const drafterState = {
+			basics: [0, 1, 2, 3, 4],
+			cardsInPack: booster.map((_, idx) => idx + 5),
+			picked: this.cards.map((_, idx) => idx + 5 + booster.length),
```

Comment:
Related to the previous comment, and if I understand this correctly, by appending the booster oracle ids to the end of `cardOracleIds` we can skip this 'translation' and built this array incrementally too (`this.cards` can simply store `oracle_id`'s directly)

Reply:
Yeah that's a great idea, I'll make the changes to do it. It'll also allow us to save some computation by using the same indices for booster as the cards we add to seen. Could also work to dedupe the array if you care about the small improvements there.